airflow-core.yaml

```diff
@@ -0,0 +1,149 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - mysql
+      - openssl
+      - postgresql-client
+      - postgresql-dev
+      - python-3.12
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - gcc
+      - glibc-dev
+      - mariadb-connector-c-dev
+      - mariadb-dev
+      - openssf-compiler-options
+      - pkgconf-dev
+      - postgresql-dev
+      - py3.12-build-bin
+      - py3.12-pip
+      - python-3.12
+      - python-3.12-dev
+      - wolfi-base
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/airflow
+      tag: ${{package.version}}
+      expected-commit: c083e456fa02c6cb32cdbe0c9ed3c3b2380beccd
+
+  - runs: |
+      # Set required build environment variables
+      export SOURCE_DATE_EPOCH=315532800
```

Comment:
Set this in the package environment with an explanation on why it's needed

Reply:
Done